frontend/src/container/NotificationWall.jsx

```diff
@@ -161,6 +162,20 @@ export class NotificationWall extends React.Component {
     return { icon, text, url }
   }
 
+  handleClickMarkAllAsRead = async () => {
+    const { props } = this
+
+    const fetchAllPutNotificationAsRead = await props.dispatch(putAllNotificationAsRead(props.user.userId))
+    switch (fetchAllPutNotificationAsRead.status) {
+      case 204: {
```

Comment:
Braces are unnecessary here, but harmless and I don't mind them.

Reply:
Changed